daemon/config/options.go

```diff
@@ -0,0 +1,21 @@
+package config // import "github.com/docker/docker/daemon/config"
+
+// WithDefaults initializes platform-agnostic default values for the Config.
+func WithDefaults(cfg *Config) {
```

Comment:
If this is only used by `New()` (and unconditionally at that), does it need to be exported?

Reply:
Ah, yes, I guess I can un-export it